README.md

```diff
@@ -1,2 +1,54 @@
 # cosmogony
-simplify osm administrative boundaries handling
+
+:construction::warning: This is a work in progress, for the moment we have ideas but not code :stuck_out_tongue_winking_eye: :warning::construction:
+
+##  Goals
+The goal of the project is to simplify the uses of administrative regions (admins) in [OpenStreetMap](https://www.openstreetmap.org).
+
+The OSM admins have several problems:
+ * admin_level (:construction:)
+ * no hierarchy (:construction:)
+
+We need a structured admin hierarchy to easily known that [paris](https://www.openstreetmap.org/relation/7444) is `city` in the `state` [ile de france](https://www.openstreetmap.org/relation/8649) in the `country` [france](https://www.openstreetmap.org/relation/2202162).
```

Comment:
For your needs (have no ambiguity with an address), this is not good enough.

There are two `Rue Jean Jaurès` in the `Commune` (that you’d map to `city`) of Lille, because of the merge with Lomme.

https://www.openstreetmap.org/way/34115830 https://www.openstreetmap.org/way/227296821

I don’t think that the mapping from libpostal will be enough

Reply:
Yes libpostal hierarchy cannot handle `rue jean jaures` in Lille, but we might be fine handling 99,99% ot the addresses in the worlds :wink: